guides.md

```diff
@@ -70,6 +77,17 @@ members. Skip the steps below at your discretion for low-stakes topics.
   - as needed, set up loomio threads and decisions
   - as needed, send reminders of action items
   - migrate action items into task tracker
+  
+### Joining a Jitsi Call
+
+💻 **Desktop.** Follow [`link.hypha.coop/calls`](https://link.hypha.coop/calls) to our default conference platform.
+
+📱 **Mobile**
+  1. Install the app. [Android][jitsi-android] | [iOS][jitsi-ios]
+  2. 🔗 **Visit** call URL to open app, or 📋 **copy-paste** redirected call URL into app home screen.
```

Comment:
```suggestion
  2. 🔗 **Visit** call URL to open app, or 📋 **copy-paste** redirected call URL (e.g. `https://jitsi.hypha.coop/hyphacoop`) into app home screen.
```

Reply:
Does this address your comment @ASoTNetworks ?

Reply:
Yes that looks good.